src/core/MIGRATION.md

```diff
@@ -824,6 +825,46 @@ After plugins, core is where most of the rest of the code in Kibana will exist.
+| `chrome.getUiSettingsClient`                          | `core.uiSettings`              |                                                                                                                                                |
+| `chrome.helpExtension.set`                            | `core.chrome.setHelpExtension` |                                                                                                                                                |
+| `chrome.setVisible`                                   | `core.chrome.setVisible`       |                                                                                                                                                |
+| `chrome.getInjected`                                  | --                             | Not available, we'd like to hear about your usecase.                                                                                           |
+| `chrome.setRootTemplate` / `chrome.setRootController` | --                             | Use application mounting via `core.application.register` (coming soon).                                                                        |
+| `import { recentlyAccessed } from 'ui/persisted_log'` | `core.chrome.recentlyAccessed` |                                                                                                                                                |
+| `ui/documentation_links`                              | `core.docLinks`                |                                                                                                                                                |
+| `ui/kfetch`                                           | `core.http`                    | API is nearly identical                                                                                                                        |
+| `ui/metadata`                                         | `core.injectedMetadata`        | May be removed in the future. If you have a necessary usecase, please let us know.                                                             |
+| `ui/notify`                                           | `core.notifications`           | Currently only supports toast messages. Banners coming soon.                                                                                   |
+| `ui/routes`                                           | --                             | There is no global routing mechanism. Each app [configures its own routing](/rfcs/text/0004_application_service_mounting.md#complete-example). |
+
+_See also: [Public's CoreStart API Docs](/docs/development/core/public/kibana-plugin-public.corestart.md)_
+
+#### Server-side
+
+In server code, `core` can be accessed from either `server.newPlatform` or `kbnServer.newPlatform`. There are not currently very many services available on the server-side:
```

Comment:
`server.newPlatform` is still KbnServer instance or Hapi instance? AFAIK we extend only KbnServer, so this reference can confuse

Reply:
We do both:
- https://github.com/joshdover/kibana/blob/029874f3ae4ed285de131ae1f688f13f0e9cc2f9/src/legacy/server/kbn_server.js#L61
https://github.com/joshdover/kibana/blob/029874f3ae4ed285de131ae1f688f13f0e9cc2f9/src/legacy/server/core/index.ts#L30